vaadin-testbench-unit/src/main/java/com/vaadin/testbench/unit/ComponentQuery.java

```diff
@@ -564,6 +564,17 @@ public <X extends ComponentWrap<? extends T>> X id(String id) {
         return find();
     }
 
+    /**
+     * Checks if this {@link ComponentQuery} describes existing components. Same
+     * as {@code !all().isEmpty() }.
```

Comment:
```suggestion
     * Checks if this {@link ComponentQuery} describes existing components.
```

I feel the implementation detail is not needed on the javadoc.